app/ga.js

```diff
@@ -1,13 +1 @@
-(function(i, s, o, g, r, a, m) {
-    i['GoogleAnalyticsObject'] = r;
-    i[r] = i[r] || function() {
-        (i[r].q = i[r].q || []).push(arguments)
-    }, i[r].l = 1 * new Date();
-    a = s.createElement(o),
-        m = s.getElementsByTagName(o)[0];
-    a.async = 1;
-    a.src = g;
-    m.parentNode.insertBefore(a, m)
-})(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
-ga('create', 'UA-101860681-5', 'auto');// add your tracking ID here.
-// ga('send', 'pageview'); // not needed as handler triggered
\ No newline at end of file
+/** overwrite this file with your own Google Analytics file */
```

Comment:
We need to keep the previous file here in the `master` branch. With either an existing or empty file, the behaviour for implementers is the same - replace the file.

Reply:
The problem is that when one does not replace the file, or perhaps makes a mistake or typo in the path, the default behavior of the installation is to report things back to the Open Targets GA...One of the main goals of this PR is really to remove this default behavior.